examples/fabric/v4/cloudRouterConnectivity/MutliCloudConnection/README.md

```diff
@@ -0,0 +1,109 @@
+# ECX Fabric Layer2 MultiCloud Connection: FCR 2 AWS and Azure
```

Comment:
Excellent PR, Srushti! Just had a couple of Nit comments on this. Be sure to add the variables.tf with this PR and then it will be ready to go.